### ISSUE TITLE
rest in peace

### ISSUE THREAD

**Issue by grinapo** [hallyn/libresource#9]
could have been nice to have. :-(

**Comment by hallyn** [comment]
Hi,

for what it's worth, unlike cgmanager this project has not been officially closed.  It does however need some interested people with free time if anything is going to happen.

I really think the way to start this it to sit down for a night or two looking at the source for maybe 'top' and coreutils, and write down an api that seems to nicely abstract away all the proc accesses for both.

**Comment by grinapo** [comment]
Yep, in my experience such projects only work if someone actually come up with a framework or basic specification which can be debated, agrued against, sweared upon or implemented.

As a starting point we have some widely used utils, mostly procps stuff and top/htop, and these should work well under cgroups/openvz/virtuozzo/or whatever container technology we have laying around; examining their requirements (to prevent the need of completely rewriting them) could be a good start. People would pop up here around when there'll be working patches against top and ps. ;-)

(I have since met __sysdig__ which is an interesting approach to the problem: using a kernel module on syscall level, but it is not the ideal solution for the general case.)
